service/src/services/LibraryService.ts

```diff
@@ -124,15 +131,40 @@ export class LibraryService implements Service<fhir4.Library> {
     const query = extractIdentificationForQuery(args, params);
     const parsedParams = parseRequestSchema({ ...params, ...query }, LibraryDataRequirementsArgs);
 
+    // check to see if data requirements were already calculated for this Library and params
+    const dataReqsQuery: Filter<any> = {};
+    Object.entries(parsedParams).forEach(p => {
```

Comment:
Just checking my understanding of what we want to query for here... we don't necessarily need to loop over all the `parsedParams`, right? We are really just interested in the identifying parameters (`id`, `url`, `version`, `identifier`) in terms of the query itself? Looks like `parsedParams` can include fields like `check-system-vesrion` and `include-dependencies`, but doesn't look like their inclusion in the data reqs query has any effect on the result

Reply:
Hmm. I see your point and for the most part agree, but I think I am going to defer to @hossenlopp on this one since I know we want to be specific when it comes to the parameters used.

Reply:
This should work for now and set us up to work with additional params.